Speech-to-text-challenge-lab-best-practices.html

```
@@ -0,0 +1,1366 @@
+                        <li>
+                            VMインスタンスに SSH で接続します(Compute Engine の VM
+                            インスタンス一覧から [SSH] ボタンをクリック)。
+                        </li>
+                        <li>
+                            タスク1で控えたAPIキーを環境変数にセットします(新しいSSHセッションの場合は再設定が必要です)。
+                            <pre><code class="language-bash">export API_KEY="taskで取得したAPIキー"</code></pre>
+                        </li>
+                        <li>
+                            リクエスト用のJSONファイルを作成します。ファイル名は
+                            <code>request.json</code> とします。
+                            <pre><code class="language-bash">cat > request.json << EOF
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)